tests/Console/Output/OutputContextTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Console\Output;
+
+use PhpCsFixer\Console\Output\OutputContext;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Output\OutputContext
+ */
+final class OutputContextTest extends TestCase
+{
+    public function testProvidedValuesAreAccessible(): void
+    {
+        $outputContext = new OutputContext(
+            $output = new NullOutput(),
+            $dispatcher = new EventDispatcher(),
+            $width = 100,
+            $filesCount = 10
```

Comment:
I consider assigning values in conditions, in passing parameters etc to be a side effect.
please, don't rely on side effects.